tools/fiptool/tbbr_config.h

```diff
@@ -9,7 +9,7 @@
 
 #include <stdint.h>
 
-#include "uuid.h"
+#include <uuid.h>
```

Comment:
Remove the blank line at line 11.

Also, please update the copyright date to 2016-2017.